src/DiabloInterface/Gui/MainWindow.cs

```diff
@@ -177,6 +178,35 @@ void DebugMenuItemOnClick(object sender, EventArgs e)
             debugWindow.Show();
         }
 
+        void DifficultyNormalToolStripMenuItemOnClick(object sender, EventArgs e)
+        {
+            UnheckDifficultyMenuItems();
```

Comment:
I know it's a bit weird, maybe we can work out something better...
The problem I'm trying to solve with that is so that we can have different runes lists showing up when farming them in normal. Maybe for a normal run you're fine with `tal+eth` but maybe for a hell run the runner also wants to get `ral+ral`... Not too sure.
So the context menu is trying to solve is just giving the intention that it will be a normal/nm/hell run so it knows which rune list to display.